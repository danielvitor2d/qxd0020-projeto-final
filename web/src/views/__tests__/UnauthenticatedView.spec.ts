import { shallowMount } from '@vue/test-utils';
import { describe, expect, it } from 'vitest';

import UnauthenticatedViewVue from '../UnauthenticatedView.vue';

describe('UnauthenticatedView', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(UnauthenticatedViewVue);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
