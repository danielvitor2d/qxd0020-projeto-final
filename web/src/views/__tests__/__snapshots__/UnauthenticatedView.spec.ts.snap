// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UnauthenticatedView > renders correctly 1`] = `
"<div class=\\"bg-[#FEFEFE] min-h-screen flex flex-col items-center justify-center\\">
  <div class=\\"text-black text-4xl mb-6\\">
    <font-awesome-icon icon=\\"fa-lock\\"></font-awesome-icon> Não autenticado
  </div>
  <div class=\\"text-black text-xl text-center\\"> Você precisa fazer login para acessar esta página. Clique abaixo para fazer login ou criar uma conta. </div>
  <div class=\\"mt-8\\">
    <router-link to=\\"/auth/login\\" class=\\"bg-green-500 hover:bg-green-600 text-white px-4 py-2 rounded-md text-xl mr-4\\">Login</router-link>
    <router-link to=\\"/auth/registration\\" class=\\"bg-blue-500 hover:bg-blue-600 text-white px-4 py-2 rounded-md text-xl\\">Criar uma conta</router-link>
  </div>
</div>"
`;
